plugins/woocommerce/src/Admin/PluginsHelper.php

```diff
@@ -532,4 +534,47 @@ public static function get_activation_status( $job_id = null ) {
 		return self::get_action_data( $actions );
 	}
 
+	/**
+	 * Show notices to connect to woo.com for unconnected store in the plugin list.
+	 *
+	 * @return void
+	 */
+	public static function maybe_show_connect_notice_in_plugin_list() {
+		global $pagenow;
+
+		if ( 'plugins.php' !== $pagenow ) {
+			return;
+		}
+
+		if ( WC_Helper::is_site_connected() ) {
+			return;
+		}
+
+		$notice_string = '';
+
+		if ( count( WC_Helper::get_local_woo_plugins() ) > 0 ) {
```

Comment:
just a note that `get_local_woo_plugins(0` will also count deactivated Woo plugin

Reply:
I think this is okay, because the goal is if for some reason, a store have a woo plugin in their plugin directory but is not connected to woocom